pipeline_alerts_email/mail.py

```diff
@@ -6,18 +6,27 @@
 
 def send_email(message, to_address):
     try:
+        html_message = message.replace("\n", "<br />")
+
+        email_html_body = (
+            f"<p>{html_message}</p>"
+            '<p><strong>Ta kontakt med oss på <a href="https://app.slack.com/client/T6W3G5A4C/C01DE13PLDP">Slack</a> eller dataplattform@oslo.kommune.no dersom du trenger hjelp.</strong></p>'
+        )
+
         # Spec for "EMAIL API" from Team økonomi:
         # https://developer.oslo.kommune.no/katalog/api/31/introduction
         payload = {
             "mottakerepost": [
                 to_address,
+            ],
+            "epostbcc": [
                 "simen.heggestoyl@origo.oslo.kommune.no",  # WIP
                 "jon.morten.kristiansen@origo.oslo.kommune.no",  # WIP
             ],
             "avsenderepost": "dataplattform@oslo.kommune.no",
             "avsendernavn": "Dataplattform",
             "emne": "Feil ved kjøring av pipeline",
-            "meldingskropp": message.replace("\n", "<br />"),
+            "meldingskropp": email_html_body,
```

Comment:
Burde vi tatt oss bryet med å lage en ren tekstvariant også (hvis API-et støtter det)?

Reply:
Har kun ett felt for body, `meldingskropp` 😞 

https://gatekeeper.k8s.oslo.kommune.no/api/api-catalog/apis/31/spec

Reply:
> Har kun ett felt for body, `meldingskropp` disappointed
> 
> https://gatekeeper.k8s.oslo.kommune.no/api/api-catalog/apis/31/spec

Ok. :(